service/index/reader.go

```diff
@@ -120,13 +122,18 @@ func (r *Reader) Values(height uint64, paths []ledger.Path) ([]ledger.Value, err
 				if err != nil {
 					return fmt.Errorf("could not retrieve payload (path: %x): %w", path, err)
 				}
+
 				values[i] = payload.Value()
 				return nil
 			})
 			return err
 		})
 	}
 	err = g.Wait()
+
+	// Temporary log line to compare execution times of individual GetRegisterValues calls
+	fmt.Printf("index reader values processing time: %s\n", time.Since(t))
```

Comment:
I think a log.debug() would be more appropriate vs a printf

Reply:
Logging the height along with time to get register value may also be useful since we're trying to figure out why/what is going on with certain calls taking longer than others